scipp-imaging-2D.ipynb

```diff
@@ -9,433 +9,902 @@
     "Before starting you must:\n",
     "- Ensure that `scipp` and `mantid` are on your `PYTHONPATH`.\n",
     "- Generate the `config.py` file using `make_config.py`. Refer to the `README.md` or `python make_config.py --help` for information.\n",
-    "- Install dependency `fabio`: `conda install fabio` (used for image loading)\n",
+    "- Install dependencies : `conda install fabio tifffile` (used for image handling)\n",
     "\n",
     "For `scipp` and `mantid` follow instructions at: https://scipp.readthedocs.io/en/latest/getting-started/installation.html.\n",
     "\n",
-    "Converted to use scipp and notebook from [this file](https://git.esss.dk/testbeamline/gp2/blob/1c69213b1124982bbbe762da9c6c6457a49f2a92/reduce.py) by Dimitar Tasev on 2020-01-13"
+    "Converted to use scipp and notebook from [this repository](https://git.esss.dk/testbeamline/gp2/-/tree/stressexperiment).\n",
```

Comment:
Should we in addition explain what this notebook experiment is doing anyway? I think this script will certainly replace what came before so maybe increasing justification for this. Note it's finding the location of each Bragg edge as a function and using that information to measure the strain in the sample against the unstrained location. Maybe I can draft something?